placekey/library/js/h3-js.umd.js

```diff
@@ -0,0 +1,2 @@
e){var r=b(A),i=r[0],t=r[1],o=b(e);return M(n.getH3UnidirectionalEdge(i,t,o[0],o[1]))},A.getOriginH3IndexFromUnidirectionalEdge=function(A){var e=b(A);return M(n.getOriginH3IndexFromUnidirectionalEdge(e[0],e[1]))},A.getDestinationH3IndexFromUnidirectionalEdge=function(A){var e=b(A);return M(n.getDestinationH3IndexFromUnidirectionalEdge(e[0],e[1]))},A.h3UnidirectionalEdgeIsValid=function(A){var e=b(A);return Boolean(n.h3UnidirectionalEdgeIsValid(e[0],e[1]))},A.getH3IndexesFromUnidirectionalEdge=function(A){var r=b(A),i=r[0],t=r[1],o=e._calloc(2,f);n.getH3IndexesFromUnidirectionalEdge(i,t,o);var a=I(o,2);return e._free(o),a},A.getH3UnidirectionalEdgesFromHexagon=function(A){var r=b(A),i=r[0],t=r[1],o=e._calloc(6,f);n.getH3UnidirectionalEdgesFromHexagon(i,t,o);var a=I(o,6);return e._free(o),a},A.getH3UnidirectionalEdgeBoundary=function(A,r){var i=e._malloc(u),t=b(A);n.getH3UnidirectionalEdgeBoundary(t[0],t[1],i);var o=U(i,r);return e._free(i),o},A.h3Distance=function(A,e){var r=b(A),i=r[0],t=r[1],o=b(e);return n.h3Distance(i,t,o[0],o[1])},A.h3Line=function(A,r){var i=b(A),t=i[0],o=i[1],a=b(r),u=a[0],l=a[1],c=n.h3LineSize(t,o,u,l);if(c<0)throw new Error("Line cannot be calculated");var s=e._calloc(c,f);n.h3Line(t,o,u,l,s);var w=I(s,c);return e._free(s),w},A.experimentalH3ToLocalIj=function(A,r){var i,o=e._malloc(w),f=n.experimentalH3ToLocalIj.apply(n,b(A).concat(b(r),[o])),a={i:e.getValue(i=o,"i32"),j:e.getValue(i+t,"i32")};switch(e._free(o),f){case 0:return a;case 1:throw new Error("Incompatible origin and index.");case 2:default:throw new Error("Local IJ coordinates undefined for this origin and index pair. The index may be too far from the origin.");case 3:case 4:case 5:throw new Error("Encountered possible pentagon distortion")}},A.experimentalLocalIjToH3=function(A,r){if(!r||"number"!=typeof r.i||"number"!=typeof r.j)throw new Error("Coordinates must be provided as an {i, j} object");var i,o,a,u=e._malloc(w),l=e._malloc(f);a=(o=r).j,e.setValue(i=u,o.i,"i32"),e.setValue(i+t,a,"i32");var c=n.experimentalLocalIjToH3.apply(n,b(A).concat([u],[l])),s=v(l);if(e._free(u),e._free(l),0!==c)throw new Error("Index not defined for this origin and IJ coordinates pair. IJ coordinates may be too far from origin, or a pentagon distortion was encountered.");return s},A.hexArea=function(A,e){switch(d(A),e){case g.m2:return n.hexAreaM2(A);case g.km2:return n.hexAreaKm2(A);default:throw new Error("Unknown unit: "+e)}},A.edgeLength=function(A,e){switch(d(A),e){case g.m:return n.edgeLengthM(A);case g.km:return n.edgeLengthKm(A);default:throw new Error("Unknown unit: "+e)}},A.numHexagons=function(A){d(A);var e=E(n.numHexagons(A)),r=e[0],i=e[1];return i?i*Math.pow(2,32)+r:r},A.getRes0Indexes=function(){var A=n.res0IndexCount(),r=e._malloc(f*A);n.getRes0Indexes(r);var i=I(r,A);return e._free(r),i},A.getPentagonIndexes=function(A){d(A);var r=n.pentagonIndexCount(),i=e._malloc(f*r);n.getPentagonIndexes(A,i);var t=I(i,r);return e._free(i),t},A.degsToRads=G,A.radsToDegs=x});
```

Comment:
This is already being imported by the H3 module. It should depend on it, and not duplicate an extra library.

Reply:
Initially I implemented it like that. Started to have issues with the ci. At some point I though that maybe we wanted totally independent modules + I was having issues with the CI, so I grouped everything together. In some way I like it depending on the other module, so we don't have duplicated codes to maintain.